lib/babe/babe.go

```diff
@@ -274,7 +274,7 @@ func (b *Service) Resume() error {
 		return nil
 	}
 
-	b.pause = make(chan struct{})
+	b.pause = make(chan struct{}, 100)
```

Comment:
i think this one could be `make(chan struct{}, 1)`

Reply:
updated.